material-popup-menu/src/test/java/com/github/zawadz88/materialpopupmenu/MaterialPopupMenuBuilderTest.kt

```diff
@@ -274,7 +277,9 @@ class MaterialPopupMenuBuilderTest {
         val secondPopupMenuItem = secondItem as MaterialPopupMenu.PopupMenuCustomItem
         assertEquals("Invalid item layout ID", CUSTOM_ITEM_LAYOUT, secondPopupMenuItem.layoutResId)
         assertEquals("Invalid item callback", customCallback, secondPopupMenuItem.callback)
-        assertEquals("Invalid item view bound callback", customViewBoundCallback, secondPopupMenuItem.viewBoundCallback)
+
+        secondPopupMenuItem.viewBoundCallback.invoke(view)
```

Comment:
This feels like something that deserves a separate test to me since you're making and additional step in the verification section of the test (the one in `//then`). 
A separate test for this with a single item and `secondPopupMenuItem.viewBoundCallback.invoke(view)` in the `//when` section would be more appropriate I think as it would indicate more clearly which part is being tested (the code for constructing the popup menu should be in `//given` section then as it's a part of the test preparation).

Reply:
Good idea. I've added one.